packages/miniflare/src/index.ts

```diff
@@ -1374,9 +1365,8 @@ export class Miniflare {
 			sharedOptions: sharedOpts.core,
 			allWorkerRoutes,
 			/*
-			 * - if Workers + Assets project but NOT Vitest, point to Router Worker or
-			 *   Proxy Worker depending on whether experimental flag `unsafeEnableAssetsRpc`
-			 *   was set
+			 * - if Workers + Assets project but NOT Vitest, point to the (assets) RPC Proxy
+			 *   Worker
 			 * - if Vitest with assets, the self binding on the test runner will point to
```

Comment:
Should the self binding point to the RPC proxy?

Reply:
I'm confused abt that comment as well...I need to check with Emily which is the source of truth there...the comment or the code

Reply:
Specifically, this other comment seems contradictory: https://github.com/cloudflare/workers-sdk/pull/8597/files#diff-f46f54e70e03a1c9ddf8abc506eb10b5ff24424bc806f8c2c245aaaa07543239R272. But I haven't poked at the internals closely, so there might be a totally legit reason why its set in one place but not the other.

Reply:
to be clear, this was the original [comment](https://github.com/cloudflare/workers-sdk/pull/8288/files#diff-9ceb732889eaca4ea99209c2d4988f82a6ed60a853000e856d47fdcb687d67f2L1313-L1314), and this was the original [code](https://github.com/cloudflare/workers-sdk/pull/8288/files#diff-9ceb732889eaca4ea99209c2d4988f82a6ed60a853000e856d47fdcb687d67f2L1320). So this is pre our rpc work

Reply:
spoke with Emily, and turns out the code was correct, and the comment just a tad misleading. I updated the comment to correct that, and posting here a diagram for posterity, that highlights the two use cases 
<img width="1297" alt="Screenshot 2025-03-21 at 11 53 30" src="https://github.com/user-attachments/assets/57447d17-d07e-430e-b968-79b2a33c3876" />

Reply:
Makes sense! Ship it!